docs/consumer-complaint-database.md

```diff
@@ -0,0 +1,9 @@
+# Consumer Complaint Database API
+
+### A resource for searching complaints submitted to the CFPB
+
+Each week the CFPB sends thousands of consumers’ complaints about financial products and services to companies for response. Those complaints are [published](https://www.consumerfinance.gov/data-research/consumer-complaints/) on our website after the company responds or after 15 days, whichever comes first.
+
+The API allows automation of the same filtering and searching functions offered to website visitors.
+
+Detailed documentation for the search API can be found [here](https://higs4281.github.io/ccdb5-api/swagger/index.html).
```

Comment:
This should live under `cfpb`, not `higs4281` 😉 .

Could you PR that gh-pages branch to [ccdb5-api](https://github.com/cfpb/ccdb5-api/)?

Reply:
Working on this.